src/components/expanded-state/CustomGasState.js

```diff
@@ -99,7 +97,6 @@ export default function CustomGasState({ asset }) {
           theme="dark"
           validateGasParams={validateGasParams}
           marginTop={19}
-          crossChainServiceTime={getCrosschainSwapServiceTime(tradeDetails)}
```

Comment:
can you elaborate on why this was removed?

Reply:
aka CustomGasState is only used for same chain transactions now.